doc/usermanual.md

```diff
@@ -83,7 +83,7 @@ Returns the current log level, in a json payload with a single attribute `level`
 
 ### Transactions
 The library implements a concept of transactions, in order to follow the execution flow the library follows when treating
-requests entering both from the Northbound and the Southbound.
+requests entering both from the North and the South ports of the IoT Agent.
```

Comment:
In line 90:

> Southbound requests, the user is responsible of

Reply:
Fixed  376c3e2 - replaced with *requests received on the South Port*